<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>doDBTableEditorUi</class>
 <widget class="QWidget" name="doDBTableEditorUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="tablesBox">
     <property name="title">
      <string>Tabellen</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QComboBox" name="tablesCBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTableDelete">
        <property name="text">
         <string>Löschen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTableNew">
        <property name="text">
         <string>Neu</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupTable">
     <property name="title">
      <string>Ausgewählte Tabelle</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="tableNameLabel">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tableName"/>
      </item>
      <item>
       <widget class="QLabel" name="tableDisplayNameLabel">
        <property name="text">
         <string>Anzeigename</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tableDisplayName"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnTableSave">
        <property name="text">
         <string>Speichern</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupColumns">
     <property name="title">
      <string>Spalten</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTableWidget" name="listColumns">
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="btnColumnAdd">
            <property name="text">
             <string>Hinzufügen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnColumnDelete">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Löschen</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupColumn">
     <property name="title">
      <string>Spalte</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="4">
       <widget class="QLabel" name="tableColumnTypeLabel">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QComboBox" name="columnType">
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QCheckBox" name="columnOptUnique">
        <property name="text">
         <string>Unique</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="tableColumnNameLabel">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QCheckBox" name="columnOptPrimaryKey">
        <property name="text">
         <string>Primary Key</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="columnName"/>
      </item>
      <item row="0" column="9">
       <widget class="QPushButton" name="btnColumnSave">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Speichern</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="columnDisplayName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>AnzeigeName</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QCheckBox" name="columnOptNotNull">
        <property name="text">
         <string>Not Null</string>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QCheckBox" name="ckBoxDisplayColumn">
        <property name="text">
         <string>Anzeigespalte</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnTableCreate">
     <property name="text">
      <string>Tabelle in Datenbank erzeugen</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Schließen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
